README.md

```diff
@@ -195,6 +195,7 @@ Currently documentation is available for the following systems:
 - [WEHI](docs/wehi.md)
 - [WUSTL_HTCF](docs/wustl_htcf.md)
 - [XANADU](docs/xanadu.md)
+- [YORK_VIKING_SLURM](docs/york_viking_slurm.md)
```

Comment:
Seems rather clunky...

I  think `YORK_VIKING` is probabkly OK, but for the `SLURM` bit could probably make a sub profile

Reply:
Absolutely fine with this being called york_viking.